ci/release_notes.md

```diff
@@ -1,10 +1,107 @@
-## Bug Fixes
-* CRITICAL: A distributed Minio manifest would only deploy 1 Minio instance.
-* Fixed a typo in the post-deploy
 
-## New Features
-* `s3` addon command, which will automatically set your envvars and run commands
-  against `s3`.
+# Overview
+This release updates CF components to more recent versions. Most
+notably, this version brings the following related CF changes:
+* Diego now uses mTLS for communication.
+* CAPI now supports IAM Service Accounts when authenticating with
+  Google Cloud Storage
+* Scalable Syslog is now used, allowing multiple VMs to handle log
+  output.
```

Comment:
Separate the paragraph from the list below it and the header above it, with blank lines.

Also, if you can convert the lists to use `-` instead of `*` for the bullet sigil, that would be more in-line with all of our other release notes.